app/code/Magento/AsynchronousOperations/Model/Operation.php

```diff
@@ -162,6 +163,23 @@ public function setErrorCode($errorCode)
         return $this->setData(self::ERROR_CODE, $errorCode);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function getOperationKey(): ?int
+    {
+        return (int) $this->getData(self::OPERATION_KEY);
```

Comment:
seems like we should check if the `$this->getData(self::OPERATION_KEY)` is not a null, something like this.
```suggestion
        return $this->getData(self::OPERATION_KEY) === null ? (int)$this->getData(self::OPERATION_KEY) : null;
```